.github/workflows/Release-candidate-containers.yml

```diff
@@ -0,0 +1,119 @@
+        environment:
+          type: choice
+          required: true
+          options:
+          - TEST
+        values:
+          type: choice
+          required: true
+          options:
+          - test2          
+jobs:
+  build-nbs-gateway-release-candidate:
+    name: Build NBS Gateway Release Candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Release-gradle-microservice-container.yaml@main
+    with:
+      microservice_name: nbs-gateway    
+      build-new-container: true
+      dockerfile_relative_path: -f ./apps/nbs-gateway/Dockerfile .
+      environment_classifier: ${{inputs.environment}}
+    secrets:
+      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
+      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  prepare-nbs-gateway-release-candidate:
+    name: Prepare NBS Gateway Release Candidate Deployment
+    needs: build-nbs-gateway-release-candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Update-helm-charts.yaml@main
+    with:
+      microservice_name: nbs-gateway
+      values_file_with_path: charts/nbs-gateway/values_${{inputs.values}}.yaml
+      new_image_tag: ${{ needs.call-build-microservice-container-workflow.outputs.output_image_tag }}
+    secrets:
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  build-modernization-api-release-candidate:
+    name: Build Modernization API Release Candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Release-gradle-microservice-container.yaml@main
+    with:
+      microservice_name: modernization-api    
+      build-new-container: true
+      dockerfile_relative_path: -f ./apps/modernization-api/Dockerfile .
+      environment_classifier: ${{inputs.environment}}
+    secrets:
+      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
+      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  prepare-modernization-api-release-candidate:
+    name: Prepare Modernization API Candidate Deployment
+    needs: build-modernization-api-release-candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Update-helm-charts.yaml@main
+    with:
+      microservice_name: modernization-api
+      values_file_with_path: charts/modernization-api/values_${{inputs.values}}.yaml
+      new_image_tag: ${{ needs.call-build-microservice-container-workflow.outputs.output_image_tag }}
+    secrets:
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}      
+  build-elasticsearch-release-candidate:
```

Comment:
Is there always a need to create a new Elasticsearch (or even NiFi) release candidate? In many cases we can reuse the same one. 

One way to customize is with boolean inputs for each microservice. In the case of elasticsearch you would have an input like the one below. And add a new line after line 68 and after line 82 that says -- if: ${{ inputs.build-elasticsearch == 'true' }}

build-elasticsearch:
   description: 'Release a new ElasticSearch Container?'
   required: true
   type: boolean
   default: true

Reply:
implemented this, thanks